core/arch/arm/mm/core_mmu.c

```diff
@@ -571,11 +571,19 @@ static void init_mem_map(struct tee_mmap_region *memory_map, size_t num_elems)
 	assert(end <= CFG_TEE_RAM_START + CFG_TEE_RAM_VA_SIZE);
 
 	if (core_mmu_place_tee_ram_at_top(va)) {
+		bool __maybe_unused va_is_secure = true; /* any init fits */
```

Comment:
The 2 are unrelated (but from functional purpose. Yet, ok to move definition to function header.